extensions/wikia/VideosModule/Modules/VideosModuleBase.class.php

```diff
@@ -122,7 +122,6 @@ public function getVideos() {
 	protected function getCacheKey() {
 		$cacheKey = wfSharedMemcKey(
```

Comment:
If this calls `wfMemcKey`, dbname will be added as prefix. Maybe this is a more straightforward solution. What do you think @garthwebb ?